irssi/PKGBUILD

```diff
@@ -7,6 +7,7 @@ replaces=(irssi-git)
 conflicts=(irssi-git)
 pkgver=1.0.2
 pkgrel=1
+epoch=1
```

Comment:
Git for Windows does not modify `irssi` Therefore this change should be unnecessary.